core/native_type_traits.h

```diff
@@ -133,4 +133,34 @@ struct NativeTypeTraits<IDLString> : public NativeTypeTraitsBase<IDLString> {
   }
 };
 
+// FIXME(Hwansung): should be generated automatically
```

Comment:
i think this lines should be generated automatically at another files.

Reply:
I thinks so too also